docs/configuration/description.md

```diff
@@ -0,0 +1,69 @@
+Files are placed in the `.mainframer` directory.
```

Comment:
This sentence is overlapped with next paragraph and also is incorrect in case of `${HOME}/.config/mainframer`